common/static/common/templates/components/paging-header.underscore

```diff
@@ -4,8 +4,8 @@
     </div>
     <nav class="pagination pagination-compact top">
         <ol>
-            <li class="nav-item previous"><a class="nav-link previous-page-link" href="#"><i class="icon fa fa-angle-left"></i> <span class="nav-label"><%= gettext("Previous") %></span></a></li>
-            <li class="nav-item next"><a class="nav-link next-page-link" href="#"><span class="nav-label"><%= gettext("Next") %></span> <i class="icon fa fa-angle-right"></i></a></li>
+            <li class="nav-item previous"><a class="nav-link previous-page-link" href="#"><i class="icon fa fa-angle-left"></i> <span class="nav-label"><%- gettext("Previous") %></span></a></li>
```

Comment:
@cptvitamin this template only affects the assets page in studio, not the teams topic list. You're seeing it in the diff because I added HTML escaping to the interpolation we had.

That being said, I'm happy to fix this. @frrrances does this make sense to you? I've made the change that @cptvitamin suggested, but it's screwed up the styling:

![screen shot 2015-06-25 at 11 00 50 am](https://cloud.githubusercontent.com/assets/2146312/8357556/6d70692e-1b29-11e5-964b-a6eaa8e5e584.png)